apis/fluentd/v1alpha1/plugins/output/types.go

```diff
@@ -42,6 +42,8 @@ type Output struct {
 	Stdout *Stdout `json:"stdout,omitempty"`
 	// out_loki plugin
 	Loki *Loki `json:"loki,omitempty"`
+	// out_cloudwatch plugin
+	Cloudwatch *Cloudwatch `json:"cloudwatch,omitempty"`
```

Comment:
@jseutter would you help to take a look at the github action errors?

Reply:
The missing definition error should be fixed now.